### PR TITLE
messages and messages overview initial desktop layout

### PR DESCRIPTION
**Pull request by jchabrowski**

### CODE REVIEW

File: frontend/src/assets/HeroIcon/index.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import * as HeroIcons from '@heroicons/react/24/outline'
+
+export type IconName = keyof typeof HeroIcons
+
+type Props = {
+  icon: IconName
+  styles: string
```
Comment (jchabrowski) [inline frontend/src/assets/HeroIcon/index.tsx:7]: This is wacky, but it works. I dont like the idea of passing random strings to the icon, but somehow icons should receive dynamic styles to change color once light/darkmode is implemented.

File: frontend/src/components/ChatMessage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+const ChatMessage = ({ variant = 'incoming' }: Props) => {
+  const avatarBgColor = getRandomElement('color')
+
+  const bgColor = variant === 'incoming' ? 'bg-slate-400' : 'bg-cyan-400'
+
+  return (
+    <div className="flex items-end space-x-2">
+      <div
+        className={`rounded-full min-h-8 min-w-8 max-h-8 max-w-8 flex items-center justify-center overflow-hidden ${avatarBgColor}`}
```
Comment (jchabrowski) [inline frontend/src/components/ChatMessage/index.tsx:15]: I don't like the line break plus there should be some order of applying classes.

File: frontend/src/components/DetailedMessageSlab/index.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { getRandomElement } from '../../mocks'
```
Comment (jchabrowski) [inline frontend/src/components/DetailedMessageSlab/index.tsx:1]: I have to create a rule so imports can work like this `from /mocks` or `from @/mocks`

File: frontend/src/components/IconNavLink/index.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import { NavLink } from 'react-router-dom'
+import DynamicHeroIcon, { IconName } from '../../assets/HeroIcon'
+
+type Props = {
+  path: string
```
Comment (jchabrowski) [inline frontend/src/components/IconNavLink/index.tsx:5]: Paths can be random string right now. They will be safeguarded.

File: frontend/src/components/ChatMessage/index.tsx (hunk 2 above)
Comment (lrozanski) [reply frontend/src/components/ChatMessage/index.tsx:15]: https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

**Review by lrozanski — APPROVED** [review]

**Pull request merged**